src/compaction_filter.rs

```diff
@@ -197,6 +197,24 @@ impl CompactionFilterContext {
             TableProperties::from_ptr(raw)
         }
     }
+
+    pub fn key_range(&self) -> (&[u8], &[u8]) {
```

Comment:
Didn't notice the interface change. I prefer `start_key()/end_key()`.

Reply:
I chaged it because I think the two interface will always be used together. XD

Reply:
I can relate, but still interface in this library shouldn't sacrifice consistency (with RocksDB interface and data member naming) for usability.

Reply:
> interface in this library shouldn't sacrifice consistency

Yeah, I think that's the point. I will make the interfaces consistent with the underlying ones.